devbin/sync.py

```diff
@@ -41,18 +30,27 @@ def __init__(self, paths: List[Tuple[str, str]]):
         self.paths = paths
 
     async def sync_and_restart_pod(self, pod, namespace):
-        await asyncio.gather(*[
-            retry(check_shell, f'{DEVBIN}/krsync.sh {RSYNC_ARGS} {local} {pod}@{namespace}:{remote}')
-            for local, remote in self.paths])
-        await retry(check_shell, f'kubectl exec {pod} --namespace {namespace} -- kill -2 1')
+        try:
+            await asyncio.gather(*[
+                retry(check_shell, f'{DEVBIN}/krsync.sh {RSYNC_ARGS} {local} {pod}@{namespace}:{remote}')
```

Comment:
I'm confused how this can still work when you got rid of the retry function above. I think you want to remove the retry here?

Reply:
oh, yep, that's a bad rebase